### DISCUSSION TITLE
34B Satış

### DISCUSSION THREAD

**Discussion by Yagiz10** [sonsuzus/sonsuzus#530] [Soru Cevap]
Bob'un eski televizyonların satıldığı bir yere gittiğini düşünelim. Bu yerde n tane televizyon var ve i indeksli televizyonun fiyatı $ a_i $ bellar. Bazı televizyonların fiyatı negatif; yani sahipleri Bob'a bu işe yaramaz cihazları satın alması için para vermeye hazır. Bob, istediği kadar televizyon satın alabilir, ancak en fazla m televizyon taşıyabilir ve ikinci kez satışa gitmek istemiyor. Bob'un kazanabileceği maksimum para miktarını bulunuz.

**Comment by Yagiz10** [comment]
```
n = int(input("Televizyon sayısını girin (n): "))
prices = list(map(int, input("Televizyon fiyatlarını girin (a1, a2, ...): ").split()))
m = int(input("Taşıyabileceği maksimum televizyon sayısını girin (m): "))

max_earnings = sum(sorted([p for p in prices if p < 0])[:m])
print("Bob'un kazanabileceği maksimum para:", max_earnings)
```

**Reply by sonsuzus** [reply]
Hangi yapay zeka çözdü? chat gpt? gemini? copilot?
